<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMultiWindow</class>
 <widget class="QMainWindow" name="TestMultiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestMultiWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actWidgetInsite"/>
   <addaction name="actWidget"/>
   <addaction name="actWindowInsite"/>
   <addaction name="actWindow"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actWindowInsite">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/2.ico</normaloff>:/images/2.ico</iconset>
   </property>
   <property name="text">
    <string>嵌入式MainWindow</string>
   </property>
   <property name="toolTip">
    <string>嵌入式MainWindow</string>
   </property>
  </action>
  <action name="actWidgetInsite">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/100.bmp</normaloff>:/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>嵌入式Widget</string>
   </property>
   <property name="toolTip">
    <string>Widget嵌入式窗体</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/122.bmp</normaloff>:/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actWindow">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/3.ico</normaloff>:/images/3.ico</iconset>
   </property>
   <property name="text">
    <string>独立MainWindow窗口</string>
   </property>
   <property name="toolTip">
    <string>独立MainWindow窗口</string>
   </property>
  </action>
  <action name="actWidget">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/804.bmp</normaloff>:/images/804.bmp</iconset>
   </property>
   <property name="text">
    <string>独立Widget窗口</string>
   </property>
   <property name="toolTip">
    <string>新建Widget独立窗口</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>TestMultiWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
